packages/toolbox-cosmos/package.json

```
@@ -1,6 +1,8 @@
 {
   "author": "thorswap",
   "dependencies": {
+    "@cosmjs/amino": "^0.30.1",
+    "@cosmjs/crypto": "^0.30.1",
```

```suggestion
    "@cosmjs/amino": "0.30.1",
    "@cosmjs/crypto": "0.30.1",
```

For `dependencies` we should use exact versions to be sure what version we are bundling :)